deps/build.log

```diff
@@ -0,0 +1,4 @@
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
```

Comment:
You seem to have added this file by mistake. Please delete it.

Reply:
Of course.